appengine/standard_python3/django/polls/templates/polls/detail.html

```diff
@@ -1,18 +1,18 @@
-<!--
- Copyright 2021 Google LLC
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--->
+{#
```

Comment:
What sorcery is this?

Reply:
This turns the html comment into a jinja comment, which means the licence text isn't on the rendered page source.

I'm not 100% sure if this is a feature or a regression, and as this PR has been in flight for a while, I can't recall why I made this change; if it was a bot suggestion, and auto formatting, or me being clever.